ui/state.go

```diff
@@ -9,6 +12,9 @@ import (
 type states struct {
 	loading bool // true if the window is in the middle of an operation that cannot be stopped
 	dialog  bool // true if the window dialog modal is open
+
+	txAuthor        *dcrlibwallet.TxAuthor
```

Comment:
these values are not booleans for determining what the wallet is displaying. You should probably create a key for these in the Window struct and update them there.